src/Microsoft.IdentityModel.JsonWebTokens/Microsoft.IdentityModel.JsonWebTokens.csproj

```diff
@@ -27,11 +27,6 @@
     <ProjectReference Include="..\Microsoft.IdentityModel.Tokens\Microsoft.IdentityModel.Tokens.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
```

Comment:
Why is this different from other csproj changes?

Reply:
These should all be transitive deps already from src/Microsoft.IdentityModel.Tokens/Microsoft.IdentityModel.Tokens.csproj